### PR TITLE
RK-10206 - Update breakpoints-conditional.md

### PR DESCRIPTION
**Pull request by ElinorSwery**

### CODE REVIEW

File: docs/breakpoints-conditional.md [hunk 1]
```diff
@@ -37,12 +37,17 @@ Add a Condition by right click on a breakpoint and choose the 'Edit' option. A r
...
-
-Seting time-to-live on a breakpoint places limits on the life span of the breakpoint. The breakpoint will disable automatically past the configured limit
-
-<iframe src="https://player.vimeo.com/video/373492033?color=af6bd6&title=0&byline=0&portrait=0" width="640" height="400" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
 
 **Keep in mind:** 
 1. The supported syntax may be different from your developing language and it can be confusing - just make sure you use the supported operators above.  
-2. A condition can be invalid due to typos and wrong variables, check that your variables written correctly. 
\ No newline at end of file
+2. A condition can be invalid due to typos and wrong variables, check that your variables written correctly.
```
Comment (ElDuderinos) [inline docs/breakpoints-conditional.md:43]: Maybe fix the English here? "check that your variables written correctly."

**Review by ElDuderinos — CHANGES_REQUESTED** [review]

File: docs/breakpoints-conditional.md (hunk 1 above)
Comment (ElinorSwery) [reply docs/breakpoints-conditional.md]: Thanks for picking this up. Corrected now

**Review by ElDuderinos — APPROVED** [review]

**Pull request merged**